ci/plans/ceramic.cue

```diff
@@ -157,9 +176,19 @@ dagger.#Plan & {
 							TAGS:           _tags + _extraTags + ["qa"]
 						}
 					}
+					qa: utils.#ECR & {
```

Comment:
We'll likely have to use a different tag than `qa` here. You could change the block tags to `qa_ceramic` and `qa_composedb` or something like that.

Reply:
Or you might actually be able to just take the tags out, will have to test that. I can help with that when you have time to look at this.

Reply:
Yeah, I'm not 💯 what this part is doing tbh. It's just adding the `qa` tag for `dev` images?
I feel like the env tags shouldn't be required/used.

Reply:
Updated and tested locally